### DISCUSSION TITLE
interactive debugger?

### DISCUSSION THREAD

**Discussion by dz0** [brython-dev/brython#2141] [Ideas]
Hi, 

I see brython has error reporting https://brython.info/static_doc/en/test.html
but does it support pdb style debugging?
Or even better -- modern GUI style (maybe like in [VSCode for web](https://code.visualstudio.com/docs/python/python-web))?

**Comment by PierreQuentel** [comment]
You can use `pdb` in Brython, but the user experience is not nice, because inputs are implemented with browser prompts.
